package.json

```diff
@@ -118,11 +118,11 @@
   "prettier": "@nordicsemiconductor/asset-tracker-cloud-code-style/.prettierrc",
   "dependencies": {
     "@aws-lambda-powertools/metrics": "1.12.1",
-    "@hello.nrfcloud.com/proto": "5.5.4",
+    "@hello.nrfcloud.com/proto": "5.5.10",
```

Comment:
Unrelated change. Revert.

Reply:
We need to upgrade `@hello.nrfcloud.com/proto` to use the same version of `@sinclair/typebox`